storage/innobase/log/log0recv.cc

```diff
@@ -3950,8 +3921,8 @@ bool meb_scan_log_recs(
         }
 #endif /* UNIV_DEBUG && HAVE_ASAN */
         more_data =
-            recv_sys_add_to_parsing_buf(log_block, scanned_lsn) || more_data;
->>>>>>> mysql-8.4.0
+            recv_sys_add_to_parsing_buf(log_block, scanned_lsn, data_len) ||
```

Comment:
Do u know the purpose of data_len? Should we also pass to recv_sys_parse_byte_by_byte()?

Reply:
Okay, this is also present in earlier versions (PXB 8.3/8.2, etc.), but to me, it doesn't make much sense.
Caller also passes log_block_get_data_len(block) as data_len.

There is no need to do an investigation now.

Closing the conversation for now.